hat/assets/js/apps/Iaso/domains/projects/components/CreateEditProjectDialog.tsx

```diff
@@ -55,6 +56,11 @@ const useStyles = makeStyles(theme => ({
         zIndex: -10,
         opacity: 0,
     },
+    '@global': {
+        '.MuiAutocomplete-listbox': {
+            maxHeight: '250px !important',
+        },
+    },
```

Comment:
This will probably impact all the dorpdown, if you want to do that, you should do it in hat/assets/js/apps/Iaso/styles/index.js, where we are grouping all global styles overrides